cfme/automate/explorer.py

```diff
@@ -266,7 +267,7 @@ def is_enabled(self):
         return self.enabled
 
     @classproperty
-    def default(cls):
+    def default(self, cls):
```

Comment:
A `@classproperty`'s signature should be `def propertyname(cls)`, why the "self"? I'd expect this to TypeError now.